### PR TITLE
revamp public pages

### PR DESCRIPTION
**Pull request by matallo**
36a28721-aaf9-498f-b824-e28c7fe95132
c5b03c85-5142-4184-bd9e-a5955698da83

CR @CartoDB/design

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5369/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5370/))

**Comment by matallo** [comment]
@javierarce @viddo would you give one final CR? thanks in advance

**Comment by carlostallon** [comment]
Preparing a mockup with some design changes to match latest designs for public dataset and data library datasets

**Comment by carlostallon** [comment]
Design changes:
- The subnav in white an all the content in f9f9f9
- main action inside subnav should be blue
- remove grey line to separate secondary actions "like" "fullscreen" and "social links"
- The margin from white subnav to title is 60px
- remove if possible the "also in cartodb" section in disqus module

public - map login - comments copy

File: app/views/admin/pages/shared/_like_button.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<button class="js-likes"
+<a href="#/like" class="js-likes"
```
Comment (viddo) [inline app/views/admin/pages/shared/_like_button.html.erb]: What's the benefits of having a link with a non-working href instead of a button?

**Comment by viddo** [comment]
cute,  :+1:  except for the change to `<a>` instead of `<button>` which I'm not totally convinced of

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5379/))

File: app/views/admin/pages/shared/_like_button.html.erb (hunk 1 above)
Comment (matallo) [inline app/views/admin/pages/shared/_like_button.html.erb]: when not logged in I need this button to redirect to /login; I find it uglier leaving the button then capturing the event and redirecting by JS

Comment (viddo) [inline app/views/admin/pages/shared/_like_button.html.erb]: But where is that redirect done then?

Comment (matallo) [inline app/views/admin/pages/shared/_like_button.html.erb]: view button https://github.com/CartoDB/cartodb/blob/4013-public_map/lib/assets/javascripts/cartodb/common/views/likes/view.js#L18-L41

Comment (viddo) [inline app/views/admin/pages/shared/_like_button.html.erb]: Aha, now I see what you mean. OK, fair enough, thx for clarifying :+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5382/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5383/))

**Comment by javierarce** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5390/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5394/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5395/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5398/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5401/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5404/))

**Comment by matallo** [comment]
retest this, please

**Comment by matallo** [comment]
retest this, please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5415/))

**Comment by matallo** [comment]
retest this, please

**Comment by matallo** [comment]
retest this, please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5426/))

**Pull request merged**